src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,88 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+import React, { useState } from 'react';
+import { Select } from '../../types/Select';
+
+interface Props {
+  onSelect: (val: Select) => void;
+  onInput: (value?: string) => void;
+}
+
+export const TodoFilter: React.FC<Props> = ({ onSelect, onInput }) => {
+  const [currentSelect, setCurrentSelect] = useState<Select>(Select.All);
```

Comment:
I suggest move this state into App and use for get filtered todos, instead of filteredTodos state